src/CoreShop2VueStorefrontBundle/Document/Category.php

```diff
@@ -34,8 +36,7 @@ class Category
 
     /** @ES\Property(type="integer", name="product_count") */
     public $product_count;
-
-    /** @ES\Property() */
+    
```

Comment:
This is incorrect since it omits the data from the index. Adding `@ES\Embedded` fails for unknown reasons, might need to upgrade the ES bundle.